packages/app/background-jobs-common/test/dependencyMocks.ts

```diff
@@ -33,24 +28,20 @@ export class DependencyMocks {
       transactionObservabilityManager: {
         start: vi.fn(),
         stop: vi.fn(),
-        // biome-ignore lint/suspicious/noExplicitAny: it's okay
```

Comment:
I honestly have no idea why it was complaining in https://github.com/lokalise/shared-ts-libs/pull/226 but not here